firefox_ui_tests/remote/security/test_green_larry.py

```diff
@@ -0,0 +1,107 @@
+        finally:
+            FirefoxTestCase.tearDown(self)
+
+    def test_green_larry(self):
+        with self.marionette.using_context('content'):
+            self.marionette.navigate(self.url)
+
+        # Get the information from the certificate
+        cert = self.browser.tabbar.tabs[0].certificate
+        address = self.security.get_address_from_certificate(cert)
+
+        # Check the identity popup label displays
+        org_label = self.identity_popup.organization_label
+        country_label = self.identity_popup.country_label
+        self.assertEqual(org_label.get_attribute('value'), cert['organization'])
+        self.assertEqual(country_label.get_attribute('value'), '(' + address['country'] + ')')
+
+        # Check the favicon
+        favicon = self.browser.navbar.locationbar.favicon
+        self.wait_for_condition(lambda _: favicon.get_attribute('hidden') == 'false')
+
+        # Check the identity popup box
+        identity_box = self.identity_popup.box
+        self.assertEqual(identity_box.get_attribute('className'), 'verifiedIdentity')
+
+        identity_box.click()
+
+        # Check the idenity popup doorhanger
+        doorhanger = self.identity_popup.popup
+        self.assertEqual(doorhanger.get_attribute('className'), 'verifiedIdentity')
+
+        # TODO: uncomment code when bug 1134872 lands
+        # Check that the lock icon is visible
+        # lock_icon = self.identity_popup.encryption_icon
+        # css_info_lock_icon = lock_icon.value_of_css_property('list-style-image')
+        # self.assertNotEqual(css_info_lock_icon, 'none')
+
+        # Check the host name
+        # Bug 443116 - Larry strips the 'www.' from the host name using the eTLDservice.
+        # This is expedted behavior.
+        host = self.identity_popup.host
+        common_name = self.marionette.execute_script("""
+          return Services.eTLD.getBaseDomainFromHost(arguments[0]);
+        """, script_args=[cert['commonName']])
+        self.assertEqual(host.get_attribute('textContent'), common_name)
+
+        # Check the organization name
+        owner = self.identity_popup.owner
+        self.assertEqual(owner.get_attribute('textContent'), cert['organization'])
+
+        # Check the owner location string 
+        owner_location = self.identity_popup.owner_location
+        location = self.browser.get_property('identity.identified.state_and_country')
+        location = location.replace('%S', address['state'], 1).replace('%S', address['country'])
+        location = address['city'] + '\n' + location
+        self.assertEqual(owner_location.get_attribute('textContent'), location)
+
+        # Check the verifier
+        verifier = self.identity_popup.verifier
+        l10n_verifier = self.browser.get_property('identity.identified.verifier')
+        l10n_verifier = l10n_verifier.replace('%S', cert['issuerOrganization'])
+        self.assertEqual(verifier.get_attribute('textContent'), l10n_verifier)
```

Comment:
Same here and for the following. Get rid of the extra variables as much as possible.